cloud/scope/powervs_cluster.go

```diff
@@ -1987,26 +1988,14 @@ func (s *PowerVSClusterScope) ReconcileLoadBalancers() (bool, error) {
 			continue
 		}
 
-		// check VPC load balancer exist in cloud
-		loadBalancerStatus, err := s.checkLoadBalancer(loadBalancer)
```

Comment:
why is this removed?

Reply:
`checkLoadBalancer` checks load balancer by the user provided name, which we already do in prev block [here](https://github.com/kubernetes-sigs/cluster-api-provider-ibmcloud/blob/6fc82480e3361b2658699075690765dc66966c1f/cloud/scope/powervs_cluster.go#L1963) by recently added code via [this](https://github.com/kubernetes-sigs/cluster-api-provider-ibmcloud/commit/3edcccbf57561ee7b6498b7f56fcf304bb9f0c19).

Reply:
`s.GetLoadBalancerID(loadBalancer.Name)` just checks only from status not from cloud!

Reply:
Sry, did not look properly, reverted the changes.